code/Task_List/app/src/main/java/aleksandr_logunov/task_list/ui/task/TaskViewModel.kt

```diff
@@ -18,9 +18,9 @@ class TaskViewModel : BaseViewModel(), ITaskViewModel {
     private var _task: ITask? = null
 
     override fun save(title: String, description: String, onSaved: OnSaved?) {
-        _task.let {
+        _task?.let {// use let like variable?.let {}
```

Comment:
There are two different method calls based on whether _task is null or not with the same behaviour. I've used ?.let so there is no need in creating a new variable for storing task cause it can be changed in runtime. 
I've added some comments about it.